drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -87,6 +87,12 @@
 
 #define IS_EMPTY(priv) (priv->type == MMCSD_CARDTYPE_UNKNOWN)
 
+#if CONFIG_MMCSD_MULTIBLOCK_LIMIT == 0
+# define MMCSD_MULTIBLOCK_LIMIT  UINT_MAX
```

Comment:
Since it is used with variables of `ssize_t` maybe better to define this to
```suggestion
# define MMCSD_MULTIBLOCK_LIMIT  SSIZE_MAX
```
?

Reply:
Sure.